integrations/docker/images/stage-2/chip-build-nuttx/Dockerfile

```diff
@@ -0,0 +1,41 @@
+ARG VERSION=1
+FROM ghcr.io/project-chip/chip-build:${VERSION}
+LABEL org.opencontainers.image.source https://github.com/project-chip/connectedhomeip
+
+# ------------------------------------------------------------------------------
+# Install prerequisites
```

Comment:
are any of these pre-requisites require to compile for nuttx? Specifically wondering about genromfs, however maybe others as well.

why do we need i386 architecture?

Reply:
Should I move the apt install -y genromfs xxd into integrations/docker/images/base/chip-build/Dockerfile?